CHANGELOG.md

```diff
@@ -643,16 +645,16 @@ To fix, when you try to change immutable fields they will throw an error in the
 
 ### Enhancements:
 - Improved lookup in data_sources by using filters
-- Improved tests duration by moving steps from data_source test files in the corresponding resource test files
-- Added workflow to run tests from GitHub actions
+- Improved tests duration by moving steps from data_source test files in the corresponding resource test files 
+- Added workflow to run tests from GitHub actions 
 - Split tests with build tags
 - Improve http client performance and timeouts
 
-### Documentations:
+### Documentations: 
```

Comment:
@cristiGuranIonos these look like some new white spaces, not sure if you actually wanted to add these. Anyway, it doesn't matter.